tests/PyPI-pytest/run_all.py

```diff
@@ -383,7 +388,8 @@ def main():
                     commands=[
                         "python -m pip install -U %s"
                         % os.path.join(dist_dir, os.listdir(dist_dir)[0]),
-                        "python -c print(getattr(__import__('%s'),'__compiled__','__uncompiled_version__'))"
+                        # use triple quotes for linux
+                        """python -c "print(getattr(__import__('%s'),'__compiled__','__uncompiled_version__'))" """
```

Comment:
if that is all it took, it makes me feel stupid, however it *ought *to be all it takes. :)